src/app.jsx

```diff
@@ -36,12 +36,43 @@ import {
     usageStartPolling,
     usageStopPolling,
 } from "./libvirt-dbus.js";
+import { useObject, useEvent } from "hooks";
+import * as service from 'service.js';
 
+import {
+    updateLibvirtState,
+} from './actions/store-actions.js';
 const _ = cockpit.gettext;
 
 superuser.reload_page_on_change();
 
-class App extends React.Component {
+export const App = ({ store, name }) => {
+    const libvirtService = useObject(() => service.proxy(name), null, []);
+
+    useEvent(libvirtService, "changed", () => {
+        store.dispatch(updateLibvirtState({
+            activeState: libvirtService.exists ? libvirtService.state : 'stopped',
+        }));
+    });
+    useEvent(superuser, "changed");
+
+    const { interfaces, networks, nodeDevices, storagePools, vms } = store.getState();
+    const resources = [...interfaces, ...networks, ...nodeDevices, ...storagePools, ...vms];
+    const loadingResources = resources.some(resource => isObjectEmpty(resource));
+
+    // Show libvirtSlate component if libvirtd is not running only to users that are allowed to start the service.
+    if (((!libvirtService.exists || libvirtService.state !== 'running') && superuser.allowed) ||
```

Comment:
So that means that now privileged users don't see their user libvirt domains if system libvirt is not running. Was that on purpose? On podman we treat them independently, and show the "system podman is also available... [start]" alert/notification/empty state" in this case.

Reply:
Not at all, no change of functionality done here. We just show the empty state if libvirtd is not running only if the connected user has privilege to do something about it. That behavior did not change though.

Reply:
Podmans behavior is better now that you mention it, do you mind if I change this in a follow up?

Reply:
Sure, it's fine as a follow-up. I confirm that the current behaviour on main is the same, sorry for the noise!